examples/node/pdf2svg.js

```diff
@@ -87,56 +87,36 @@ function writeSvgToFile(svgElement, filePath) {
   });
 }
 
-// Will be using promises to load document, pages and misc data instead of
-// callback.
+// Will be using async/await to load document, pages and misc data
 const loadingTask = pdfjsLib.getDocument({
   data,
   cMapUrl: CMAP_URL,
   cMapPacked: CMAP_PACKED,
   fontExtraProperties: true,
 });
-loadingTask.promise
-  .then(function (doc) {
-    const numPages = doc.numPages;
-    console.log("# Document Loaded");
-    console.log("Number of Pages: " + numPages);
-    console.log();
+(async function () {
+  const doc = await loadingTask.promise;
+  const numPages = doc.numPages;
+  console.log("# Document Loaded");
+  console.log("Number of Pages: " + numPages);
```

Comment:
Here, and elsewhere in this file, let's change these types of strings (that contains parameter values) to template strings instead; in this case it'll become:
```javascript
  console.log(`Number of Pages: ${numPages}`);
```